components/servo/examples/winit_minimal.rs

```diff
@@ -206,6 +206,16 @@ impl ApplicationHandler<WakerEvent> for App {
                     }
                 }
             },
+            WindowEvent::Resized(new_size) => {
+                if let Self::Running(state) = self {
+                    if let Some(webview) = state.webviews.borrow().last() {
+                        let mut rect = webview.rect();
+                        rect.set_size(winit_size_to_euclid_size(new_size).to_f32());
+                        webview.move_resize(rect);
```

Comment:
>  The size of webview should be determined by embedder instead of compositor. A dialog webview might want to stick to fixed size event when the window is resized. Could you send MoveResizeWebView in simpleservo?

This comment from https://github.com/servo/servo/pull/35158#discussion_r1929474897 probably also applies to this PR. CC @wusyong

Reply:
Sorry, I somehow didn't notice yesterday that this is actually in `winit_minimal.rs` .... Resolving.